src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -352,7 +352,7 @@ public static async Task WaitForReplicationAsync(DocumentDatabase database, Time
 
         public static async Task WaitForIndexesAsync(DocumentsContextPool contextPool, DocumentDatabase database, TimeSpan timeout,
             List<string> specifiedIndexesQueryString, bool throwOnTimeout,
-            string lastChangeVector, long lastTombstoneEtag, HashSet<string> modifiedCollections)
+            string lastChangeVector, long lastTombstoneEtag, HashSet<string> modifiedCollections, long lastDocumentEtag = -1)
```

Comment:
Who is setting `lastDocumentEtag ` ?

Reply:
The ETag of the command on the current node

Reply:
I'm not seeing any code calling this?

Reply:
https://github.com/ravendb/ravendb/blob/7244540e17f585743d7f0de33992feb3485e7ef9/src/Raven.Server/Documents/Handlers/BatchHandler.cs#L521